sections/1-introduction.tex

```diff
@@ -1,21 +1,38 @@
-Advancing our understanding of the fundamental processes underpinning these complex and connected systems requires interdisciplinary research across these scientific sub-disciplines.
-At the time of writing, the NASA Heliophysics System Observatory consists of 18 missions with over 100 instruments.
+Research astrophysicists rely on software for the analysis of data which is ever-growing and increasingly complex.
+Scientists that study our nearest star, the Sun, are no different.
+Solar physics relies (mostly) on remote sensing to measure the properties of our star.
 
-Software packages to analyze data from these instruments are generally developed independently by each instrument team.
-This creates a diverse and therefore difficult data and software environment to navigate.
-Scientists conducting interdisciplinary research using multiple instruments run into problems with incompatible data formats and incompatible analysis routines written in different languages.
-Furthermore, it is difficult to reproduce the scientific results of others without open data and version-controlled open source software.
+In the past, most solar data analysis was performed using Fortran, a general-purpose, compiled programming language designed for scientific and engineering applications.
+In the 1980s, the solar community made the transition from Fortran to IDL (Interactive Data Language), a commercial and closed-source interpretive programming language.
+The community extracted several significant benefits from this transition.
+Two significant ones are that IDL is an interpretive language which enables much faster (and therefore cheaper) development and access to a large number of powerful libraries that support scientific data analysis.
+The astronomy community made the same transition around the same time which led to the release of the IDL Astronomy User's Library which allowed for significant code re-use.
+Since the transition, significant and powerful functionality has been developed as part of Solarsoft, an open-source library of integrated software libraries that provide a common data analysis environment for solar physics \citet{freeland1998}.
 
-A common, version-controlled, open source platform that provides a standard interface to data products and encourages the re-use of common functions can go a long way toward solving this problem.
-The \sunpyproj aims to provide this solution for the field of solar physics consistent with best practices \citep{Wilson:2014cka}.
+A transition to Python will provide an even greater leap in capabilities to the solar community.
+Astronomy is already making this transition (cite astropy).
+Python is an interactive, interpretive, object-oriented, portable and high-level programming language which is extensible with compiled code in lower-level languages.
+The strengths of the scientific Python environment have been enumerated in a number of other papers (cite cite cite). These include (add strengths here).
```

Comment:
....

Reply:
I'm hoping that @mbobra would write this part :)